src/frontends/tensorflow_common/src/op/segment_sum.cpp

```diff
@@ -46,6 +50,10 @@ OutputVector translate_segment_sum_op(const NodeContext& node) {
     auto indices = make_shared<v4::Range>(const_zero, num_indices, const_one, indices_type);
 
     auto emb_segment_sum = make_shared<v3::EmbeddingSegmentsSum>(data, indices, segment_ids, num_segments);
+    if (complex_type_mark) {
+        auto data = complex_type_mark->input_value(0);
+        auto emb_segment_sum = make_shared<v3::EmbeddingSegmentsSum>(data, indices, segment_ids, num_segments);
```

Comment:
it is a local variable. Finally you will return `emb_segment_sum` created above.
Create `emb_segment_sum` for regular and complex type case separately.

Reply:
so I should make one using real data and complex data and return them to their respective nodes is it ?